src/main/java/com/bgsoftware/ssboneblock/listeners/BlocksListener.java

```diff
@@ -91,7 +91,7 @@ public void onOneBlockBreak(BlockBreakEvent e) {
 
             if (blockState instanceof InventoryHolder) {
                 Inventory inventory = ((InventoryHolder) blockState).getInventory();
-                if(WorldUtils.shouldDropInventory(inventory)){
+                if(WorldUtils.shouldDropInventory(inventory.getHolder())){
```

Comment:
Just forward `(InventoryHolder) blockState` instead of using `Inventory#getHolder`